### PR TITLE
Remove PSWS-Version header as it's added after and only if we are authenticated

### PR DESCRIPTION
**Pull request by PrestaEdit**
…thenticated

| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Remove an information header api returns.
| Type?             | bug fix
| Category?         | WS
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | See below

### How to test

Perform a HTTP request against web service
- if you are not authentificated the PSWS-Version header should not be set
- if you are authentificated the PSWS-Version header should be set

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/27741)

### CODE REVIEW

**Review by Progi1984 — APPROVED** [review]

**Comment by matthieu-rolland** [comment]
I restarted the sanity tests, which were failing...  I'm not sure if it's related to your PR or not (I don't see how it would be related)

File: classes/webservice/WebserviceRequest.php [hunk 1]
```diff
@@ -1790,7 +1790,6 @@ protected function returnOutput()
         $this->objOutput
             ->setHeaderParams('Access-Time', (string) time())
             ->setHeaderParams('X-Powered-By', 'PrestaShop Webservice')
-            ->setHeaderParams('PSWS-Version', _PS_VERSION_)
```
Comment (matks) [inline classes/webservice/WebserviceRequest.php:1793]: You're right 😮  this header was also added line 1800

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
@PrestaEdit I improved a little the PR body. Can you tell me if it's OK?

**Comment by PrestaEdit** [comment]
It's good for me, yes :)

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @PrestaEdit

**Pull request merged**